### DISCUSSION TITLE
Is there a Style & Composition SDXL Tiled Node?

### DISCUSSION THREAD

**Discussion by gonzalu** [cubiq/ComfyUI_IPAdapter_plus#478] [General]
Since i am really enjoying the new node, I regularly use non-square images. Is there or will there be a tiled version of this wonderful, combined node? 

Thanks Matteo

**Comment by cubiq** [comment]
it would be feasible but depends on the aspect ratio, the clip vision might not be able to catch the composition

**Comment by gonzalu** [comment]
Gotcha... just wondering. I have been using the regular advanced tiled node and seem to be getting decent results when I use the composition option. Seems to be working well so far... or at least it is not offensive :D 

thank you as always

**Comment by gonzalu** [comment]
Here is one example... 


![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/32348862/4e53b722-ba63-470c-9429-3b5eadd4c718)

![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/32348862/a3df4750-35e4-4870-9976-a209cb05b169)

![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/32348862/881ced37-1145-46b6-be45-918b2ef9ff3e)

**Reply by cubiq** [reply]
that's a very simple composition :smile:

**Reply by gonzalu** [reply]
OK OK fair enough ROTFL :D
